ChiTech/ChiPhysics/FieldFunction/export_multiple_vtk.cc

```diff
@@ -75,7 +73,7 @@ void chi_physics::FieldFunction::
 
   //============================================= Populate VTK points
   auto points = vtkSmartPointer<vtkPoints>::New();
-  int vc=0;
+  int64_t vc=0;
```

Comment:
@Naktakala I think there is another PR where very short varnames have been made more explicit. I think I remember this to be the case for "vc". If I am correct, then we should try to remain consistent throughout. Thanks